bin/terraform-docs.sh

```diff
@@ -5,7 +5,7 @@ which terraform 2>&1 >/dev/null || ( echo "terraform not available"; exit 1)
 which terraform-docs 2>&1 >/dev/null || ( echo "terraform-docs not available"; exit 1)
 
 if [[ "`terraform version | head -1`" =~ 0\.12 ]]; then
-    TMP_FILE="$(mktemp /tmp/terraform-docs-XXXXXXXXXX.tf)"
+    TMP_FILE="$(mktemp /tmp/terraform-docs-XXXXXXXXXX)"
```

Comment:
Please refer to the other pull requests to see how to format the pull request. This change by itself doesn't explain why this would generate a unique temporary file any differently.